CHANGELOG.md

```diff
@@ -2998,3 +2998,9 @@ This is a full list of proofs that have changed form to use irrelevant instance
   ```agda
   <-weakInduction-startingFrom : P i →  (∀ j → P (inject₁ j) → P (suc j)) → ∀ {j} → j ≥ i → P j
   ```
+
+* Added new functions to `Data.Vec.Relation.Binary.Equality.Setoid`
+  ```agda
+  cong-[]≔ : i p → xs ≋ ys → xs [ i ]≔ p ≋ ys [ i ]≔ p
```

Comment:
Usual custom with `CHANGELOG` is to omit inferrable arguments  from the statement... here `i` and `p`

Reply:
I removed it.